executor/src/witgen/eval_result.rs

```diff
@@ -7,8 +7,6 @@ use super::range_constraints::RangeConstraint;
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum IncompleteCause<K = usize> {
-    /// Previous value of witness column not known when trying to derive a value in the next row. Example: `x' = x` where `x` is unknown
-    PreviousValueUnknown(K),
```

Comment:
This would be a bit harder to implement now, because I changed the way we iterate over rows (see description). I think these incomplete clauses are not really used anywhere, so I think it's fine to remove.